hw_7/Task_7.2.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+// Task 2.
+
+/*
+2. У вас есть массив с ценами товаров в чеке. В консоль нужно вывести сумму всех цен и среднюю цену товара.
+  Итого: 8495 $, средняя цена товара 700 $ - пример сообщения в консоле.  
+  const prices = [64, 7556, 345, 7556, 345, 7556, 345, 7556, 433, 345, 756, 123, 942, 3112, 421, 9341, 1212, 8, 43, 41, 345, 341, 21, 321, 123];
+*/
+
+const maxAndAvgPrise = (check = []) => {
+  const total = check.reduce((total, prise) => total + prise, 0);
+  const avg = total / check.length;
+
+  return `Total price: ${total}, average product price: ${Math.round(avg)}`;
+};
+
+const prices = [
+  64, 7556, 345, 7556, 345, 7556, 345, 7556, 433, 345, 756, 123, 942, 3112, 421,
+  9341, 1212, 8, 43, 41, 345, 341, 21, 321, 123,
+];
+
+console.log(maxAndAvgPrise(prices)); 
+
+/*
+3. Напишите функцию, которая принимает на вход массив слов и возвращает отсортированный массив по по следующему критерию: количество гласных букв.
+  Массив должен быть отсортирован по возврастанию количества гласных букв в слове.
+*/
+
+function vowelCounter(word = []) {
```

Comment:
```suggestion
function vowelCounter(word = "") {
```
Исходя из твоего же кода ты получаешь строку, которую потом преобразовываешь в массив:)